core/modules/system/css/system.theme.css

```diff
@@ -585,6 +585,19 @@ tr:hover del.diffchange {
 }
 
 /**
+ * Hero blocks.
+ */
+ .hero-block {
+   background-size: cover;
+   background-repeat: no-repeat;
+   background-position: top center;
+   padding: 50px 100px;
+   text-align: center;
+   color: #ffffff;
```

Comment:
Yeah, checkbox for invert colors sounds like a nice-to-have but not required. I'm seeing the same thing on *most* heros (dark image, white text). Let's start there.